.github/workflows/ci.yml

```diff
@@ -56,3 +56,5 @@ jobs:
       # Step 8: Run Unit Tests
       - name: Run Unit Tests
         run: pnpm test
+        env:
+          AUTH_SERVER_URL: ${{ secrets.AUTH_SERVER_URL }}
```

Comment:
We shouldn't need to add an env for the test suite. We should move any init logic that depends on ENV vars to someplace that doesn't run when the tests are run.